<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabCOM">
      <attribute name="title">
       <string>Serial port</string>
      </attribute>
      <widget class="QLabel" name="lblBaudRate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>BaudRate:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblDataBits">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DataBits:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblParity">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblStopBits">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>StopBits:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblFlowControl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>FlowControl:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbBaudRate">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbDataBits">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbParity">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbStopBits">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbFlowControl">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabTOOL">
      <attribute name="title">
       <string>Tools</string>
      </attribute>
      <widget class="QLabel" name="lblMML2BIN">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MML2BIN:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblEditor">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Editor:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtMML2BIN">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEditor">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMML2BIN">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditor">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>100</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPAR2BIN">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PAR2BIN:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPAR2BIN">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPAR2BIN">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>60</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
